uk.co.compucorp.civicrm.hrcontactactionsmenu/scss/components/_actions-button.scss

```diff
@@ -0,0 +1,7 @@
+#crm-contact-actions-link {
```

Comment:
can you show me what this entire component is for?

Reply:
<img width="337" alt="screen shot 2018-01-24 at 1 33 24 pm" src="https://user-images.githubusercontent.com/6307362/35320237-364b08a8-010b-11e8-84e1-97de1c44e9ae.png">
This component is for this "Actions" button.